.github/workflows/ca-deploy-test.yaml

```diff
@@ -0,0 +1,46 @@
+name: Deploy-cerpus-test
```

Comment:
There's already a workflow with this name.

Names are shown in the Actions tab, so a human-friendly one should be chosen, i.e. `Deploy CA`